frontend/public/components/monitoring/_monitoring.scss

```diff
@@ -290,3 +290,8 @@ $tooltip-background-color: #151515;
 .co-alert-manager-config__edit-alert-routing-btn {
   margin-bottom: 10px;
 }
+
+.co-alert-manager-config__save-as-default-label {
```

Comment:
This shouldn't be necessary if you use the right checkbox styles.

Reply:
This is not for the checkbox, it is for the blue (?) icon

Reply:
> This is not for the checkbox, it is for the blue (?) icon

We should give it a better name in that case.

Reply:
I feel like we have some existing styles you can reuse for the tooltip icon.

Reply:
Hi @spadgett, seaching through css, nothing jumping out at me, what's a better name?  It matches the naming convention of the css definition above it in the .scss file.